x-pack/platform/plugins/shared/cases/server/client/alerts/get.ts

```diff
@@ -37,3 +42,41 @@ export const getAlerts = async (
     ...alert._source,
   }));
 };
+
+export const getAlertMetadataFromComments = async (
```

Comment:
We'll want to add authorization logic here https://github.com/elastic/kibana/blob/d5524b1e38ec26197a543b1f2967027a7d984425/x-pack/platform/plugins/shared/cases/server/client/cases/get.ts#L185

Potentially using https://github.com/elastic/kibana/blob/d5524b1e38ec26197a543b1f2967027a7d984425/x-pack/platform/plugins/shared/cases/server/authorization/index.ts#L239

But it'll come down to what combination of access we want the users to have from that list